### ISSUE TITLE
Add uiNamespaceVariable to spectator

### ISSUE THREAD

**Issue by badkneegrow** [acemod/ACE3#5469]
https://github.com/acemod/ACE3/blob/master/addons/spectator/ui.hpp#L20

```sqf
onLoad = QUOTE(uiNamespace setVariable [ARR_2(QUOTE(QGVAR(display)),_this select 0)];);
```
```sqf
// uiNamespace setVariable ["ace_spectator_display",_this select 0];
```

**Comment by badkneegrow** [comment]
Seems like the spectator in the master branch has this missing. But the release version spectator is different...

**Comment by SilentSpike** [comment]
Can I ask what your use case is? It was removed as part of the overhaul simply because I was no longer making use of a variable, but there's no reason it couldn't be re-added

**Comment by badkneegrow** [comment]
I want to allow admin to use admin menu and get into zeus after he gets into spectator.
https://github.com/badkneegrow/The-Mission-Framewrok/blob/master/plugins/menus/files/fnc/fn_enableUnconsciousMenu.sqf#L28

**Comment by commy2** [comment]
I'm for keeping the variable, maybe even calling an event when the display is opened.

**Comment by jonpas** [comment]
> I want to allow admin to use admin menu and get into zeus after he gets into spectator.

This should be in ACE3 Spectator itself @SilentSpike!

**Comment by SilentSpike** [comment]
Zeus? It already is. No idea what admin menu is, if it's the escape menu with debug console then it also already is.

**Comment by PabstMirror** [comment]
Closed via #5520
